sdjournal/read.go

```diff
@@ -38,7 +38,10 @@ type JournalReaderConfig struct {
 	// the array is empty, entries will not be filtered.
 	Matches []Match
 
-	MaxEntryLength uint64 // the max allowable length for an entry (default is 64K)
+	// explicitly set MaxEntryLenght to 0, which means unlimited
+	DisableMaxEntryLength bool
+	// the max allowable length for an entry (default is 64K)
+	MaxEntryLength uint64
```

Comment:
Unless a pointer is used to say the value is empty so use a default. But with a pointer struct initialization will be awful for a numeric type